dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -16,3 +16,37 @@ function onURLBlocked(base_url, callback) {
       .css("position", "absolute")
       .appendTo("body");
 }
+
+var DateHelper = {
```

Comment:
+1 for not reinventing the wheel.  If that plugin has good support let's use it.